PRASCore.jl/src/Results/Shortfall.jl

```diff
@@ -258,19 +262,19 @@ EUE(x::ShortfallResult{N,L,T,E}, t::ZonedDateTime) where {N,L,T,E} =
 EUE(x::ShortfallResult{N,L,T,E}, r::AbstractString, t::ZonedDateTime) where {N,L,T,E} =
     EUE{1,L,T,E}(MeanEstimate(x[r, t]..., x.nsamples))
 
-function NEUE(x::ShortfallResult{N,L,T,E}) where {N,L,T,E}
+function NEUE(x::ShortfallResult)  
     return NEUE(div(MeanEstimate(x[]..., x.nsamples),(sum(x.regions.load)/1e6)))
 end
 
-function NEUE(x::ShortfallResult{N,L,T,E}, r::AbstractString) where {N,L,T,E}
+function NEUE(x::ShortfallResult, r::AbstractString) 
     i_r = findfirstunique(x.regions.names, r)
     return NEUE(div(MeanEstimate(x[r]..., x.nsamples),(sum(x.regions.load[i_r,:])/1e6)))
 end
 
 function finalize(
-    acc::ShortfallAccumulator,
+    acc::ShortfallAccumulator{S},
     system::SystemModel{N,L,T,P,E},
-) where {N,L,T,P,E}
+) where {N,L,T,P,E,S<:Union{Shortfall,DemandResponseShortfall}}
```

Comment:
Nothing wrong with adding this type constraint, but know it doesn’t actually do anything since it’s the same as the constraint on S in the ShortfallAccumulator type definition